### DISCUSSION TITLE
Instruction fine tuning Flan-T5, non-english language

### DISCUSSION THREAD

**Discussion by holmbuar** [google-research/t5x#1153] [Q&A]
Are there any conventions, rules or templates when it comes to fine-tuning the Flan-T5 models in a non-English language?

How would the following example be formatted in e.g. German? Would it be a direct translation like
```
Q: Answer the following yes/no question by
reasoning step-by-step.
Could a dandelion suffer from hepatitis?
A: Hepatitis only affects organisms with livers.
Dandelions don’t have a liver. The answer is no.
```
to 
```
F: Beantworten Sie die folgende Ja/Nein-Frage mit
Schritt für Schritt argumentieren.
Könnte ein Löwenzahn an Hepatitis leiden?
A: Hepatitis befällt nur Organismen mit Lebern.
Löwenzahn hat keine Leber. Die Antwort ist nein.
```
? 

Similarly, in German, would a randomly picked FewShot pattern from [Flan templates](https://github.com/google-research/FLAN/blob/main/flan/v2/templates.py) be:

```
 FewShotPattern(
      inputs="Ist \"{text1}\" in diesem Satz dasselbe wie \"{text2}\"?\n{context}\n\n{options_}",
      targets="{answer}",
      inputs_prefix="FRAGE: ",
      targets_prefix="ANT: ",
      x_y_delimiter="\n",
      example_separator="\n"),
```
?

(Pardon the Google translated texts)

**Comment by kushmuratoff** [comment]
I also need answer, @torlarse if you found answer, share with us
